app/assets/javascripts/pageflow/editor/views/theme_view.js

```diff
@@ -0,0 +1,67 @@
+/** @api private */
+pageflow.ThemeView = Backbone.Marionette.ItemView.extend({
+  template: 'pageflow/editor/templates/theme',
+  tagName: 'li',
+
+  ui: {
+    name: '.theme_name',
+    selectButton: '.select_theme',
+    selectedRegion: '.theme_selected'
+  },
+
+  events: {
+    'click .select_theme': function(event) {
+      this.entry.configuration.set('theme_name', this.model.get('name'));
+      pageflow.updateStylesheet('theme');
+    },
+    'mouseenter': function() {
+      var that = this;
+      this.uiPreviewImage.attr('src', pageflow.editor.themes.find(function(theme) {
+        return theme.attributes.name == that.model.get('name');
+      }).get('preview_image_url'));
+      this.uiPreviewHeaderThemeName.text(that.translateThemeName());
+    },
+    'mouseleave': function() {
+      var that = this;
+      this.uiPreviewImage.attr('src', pageflow.editor.themes.find(function(theme) {
+        return theme.attributes.name == pageflow.entry.configuration.attributes.theme_name;
+      }).get('preview_image_url'));
+      this.uiPreviewHeaderThemeName.text(that.translateThemeName({
+        name: pageflow.entry.configuration.attributes.theme_name
+      }));
+    }
```

Comment:
I think this would make our code unnecessarily hard to read. We have a selection workflow and a previewing workflow accessible from here, and we would reuse a selection mixin for the preview workflow while the selection workflow uses its own code but does something which is similar to what the selection mixin does. As soon as somebody else works on this code, there is significant risk that they are confused by this.

I will see if we can work with two mixins here without much additional effort